<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogsou</class>
 <widget class="QDialog" name="Dialogsou">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialogsou</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(103, 103, 103);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>661</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Gestion de livraison</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>781</width>
       <height>621</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(207, 255, 251);

</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Ajout</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>471</width>
         <height>241</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(207, 255, 251);</string>
       </property>
       <property name="title">
        <string>Ajouter</string>
       </property>
       <widget class="QPushButton" name="Ajouter">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>90</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="num_com">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>20</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">m_myWidget = new QWidget(this); m_myWidget-&gt;setGeometry(0, 0, 300, 100); QPalette pal = palette();</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>plat</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date d'arrivee</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Numero de commande</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Total</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="date_arrivee">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>170</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
        <property name="date">
         <date>
          <year>2021</year>
          <month>2</month>
          <day>1</day>
         </date>
        </property>
       </widget>
       <widget class="QLineEdit" name="total">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>140</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Quantite</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="livreur">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>110</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="produit">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>50</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>pizza</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>spaghetti</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>salade</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Livreur</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="quantite">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>80</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Affichage</string>
      </attribute>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>471</width>
         <height>301</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>60</y>
         <width>271</width>
         <height>301</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(207, 255, 251);</string>
       </property>
       <property name="title">
        <string>Modifier</string>
       </property>
       <widget class="QLabel" name="label_37">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Numero de commande</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_36">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>210</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date d'arrivee</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Produit</string>
        </property>
       </widget>
       <widget class="QComboBox" name="produit_2">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>90</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>SAVON</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GEL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>JAVEL</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="livreur_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>150</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="total_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>180</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_38">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Quantite</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_35">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>180</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Total</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_34">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Livreur</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="date_arrivee_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>210</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
        <property name="date">
         <date>
          <year>2021</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
       </widget>
       <widget class="QLineEdit" name="num_com_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>60</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">m_myWidget = new QWidget(this); m_myWidget-&gt;setGeometry(0, 0, 300, 100); QPalette pal = palette();</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="quantite_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>120</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="modifier_2">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>260</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>380</y>
         <width>301</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(207, 255, 251);</string>
       </property>
       <property name="title">
        <string>tri</string>
       </property>
       <widget class="QComboBox" name="colone_tri">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>num_com</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>produit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>quantite</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>livreur</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>total</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>date_arrivee</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="ordre_tri">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>ASC</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DESC</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="button_tri">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>30</y>
          <width>75</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>tri</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>470</y>
         <width>271</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(207, 255, 251);</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Stat</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>380</y>
         <width>271</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(207, 255, 251);</string>
       </property>
       <property name="title">
        <string>supprimer</string>
       </property>
       <widget class="QLineEdit" name="le_supprimer">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Numéro de commande</string>
        </property>
       </widget>
       <widget class="QPushButton" name="supprimer">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>40</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
      <widget class="QComboBox" name="comboBox_rech">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>490</y>
         <width>73</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>num_cmd</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>livreur</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>date</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEdit_rech">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>490</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>gestion_reclamation</string>
    </attribute>
    <widget class="QTabWidget" name="affichage2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>771</width>
       <height>601</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(151, 196, 255);</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_11">
      <attribute name="title">
       <string>Ajout</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>471</width>
         <height>241</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(207, 255, 251);</string>
       </property>
       <property name="title">
        <string>Ajouter</string>
       </property>
       <widget class="QPushButton" name="Ajouter_2">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>90</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="id_rec">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>20</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">m_myWidget = new QWidget(this); m_myWidget-&gt;setGeometry(0, 0, 300, 100); QPalette pal = palette();</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>id_livraison</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date reclamation</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>id_rec</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="date_pan">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>110</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
        <property name="date">
         <date>
          <year>2021</year>
          <month>3</month>
          <day>30</day>
         </date>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
       <widget class="QComboBox" name="id_liv">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>50</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="descc">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>80</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_12">
      <attribute name="title">
       <string>Affichage</string>
      </attribute>
      <widget class="QTableView" name="tableView_rec">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>471</width>
         <height>301</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>60</y>
         <width>271</width>
         <height>181</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(207, 255, 251);</string>
       </property>
       <property name="title">
        <string>modifier</string>
       </property>
       <widget class="QLabel" name="label_48">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date_rec</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="date_panne2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>100</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
        <property name="date">
         <date>
          <year>2021</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
       </widget>
       <widget class="QComboBox" name="i_reccombo">
        <property name="geometry">
         <rect>
          <x>152</x>
          <y>30</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_44">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="total_5">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>70</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>id_rec</string>
        </property>
       </widget>
       <widget class="QPushButton" name="modifier_4">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>140</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_8">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>380</y>
         <width>261</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(207, 255, 251);</string>
       </property>
       <property name="title">
        <string>supprimer</string>
       </property>
       <widget class="QPushButton" name="supprimer_3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QComboBox" name="id_reccombosup">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_9">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>380</y>
         <width>120</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(207, 255, 251);</string>
       </property>
       <property name="title">
        <string>generer pdf</string>
       </property>
       <widget class="QPushButton" name="pdff">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>pdf</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
